ethereum/contracts/Application.sol

```diff
@@ -0,0 +1,7 @@
+ // SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2;
+
+// Application contains methods that all applications must implement
+abstract contract Application {
+    function submit(bytes memory _data) public virtual;
```

Comment:
To prevent devs and users confusing this method with the `submit` method on the Bridge/Broker contract, I'd rather call it `handle`.

```suggestion
    function handle(bytes memory _data) public virtual;
```
This is what we do on the Substrate side:
https://github.com/Snowfork/polkadot-ethereum/blob/main/parachain/pallets/polkaerc20-app/src/lib.rs#L110

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/75/commits/31a105e146a248e1e665be92dab4405052e072b7